Springer Link.js

```diff
@@ -89,13 +89,19 @@ function complementItem(doc, item) {
 					or @id="abstract-about-doi"][1]'
 		);
 	}
+	if (!item.language) {
+		item.language = ZU.xpathText(doc, '//meta[@name="citation_language"]/@content');
+	}	
 	if(!item.publisher) {
```

Comment:
I prefer to add a space after the `if` as you did here (but there is quite a mismatch in that file). Can we agree that we generally want that? See also https://eslint.org/docs/rules/keyword-spacing

Reply:
I believe the space is standard Zotero style, too, so yes, let's do that.

Reply:
Great!

> Unsure whether to address that retrospectively, though.

Should be possible to replace this by simple search and replace. I will try to look into this...